src/libretro/LibretroEnvironment.cpp

```diff
@@ -540,8 +540,25 @@ bool CLibretroEnvironment::EnvironmentCallback(unsigned int cmd, void *data)
     retro_vfs_interface_info* typedData = reinterpret_cast<retro_vfs_interface_info*>(data);
     if (typedData)
     {
-      // Not implemented
-      return false;
+      if (typedData->required_interface_version <= supported_vfs_version)
+      {
+        static retro_vfs_interface vfsInterface = {
+          CFrontendBridge::GetPath,
+          CFrontendBridge::OpenFile,
+          CFrontendBridge::CloseFile,
+          CFrontendBridge::FileSize,
+          CFrontendBridge::GetPosition,
+          CFrontendBridge::Seek,
+          CFrontendBridge::ReadFile,
+          CFrontendBridge::WriteFile,
+          CFrontendBridge::FlushFile,
+          CFrontendBridge::RemoveFile,
+          CFrontendBridge::RenameFile,
```

Comment:
Stay the retro API always in this flow, I think yes only want to confirm?

Seen also that it allow some other functions: https://github.com/libretro/RetroArch/blob/master/libretro-common/include/libretro.h#L1423-L1447
Makes it sense to include them too?

Reply:
Yes, this PR only implements version 1. Version 2 & 3 support is planned.

Reply:
Version 2 and 3 support has been included in this PR